src/reducers/entities/users.js

```diff
@@ -72,6 +72,31 @@ function removeProfileFromSet(state, action) {
     };
 }
 
+// This entity manipulates the entities.users.analytics object in the store
+// It contains 2 reducers:
+//     1. RECEIVED_USER_ANALYTICS
+//        - user analytics data was just received, add it to the analytics object
+//     2. LOGOUT_SUCCESS
+//        - dispatched when forceLogoutIfNecessary is called in the action after a server error
```

Comment:
I handled the LOGOUT_SUCCESS action here because I was just following the conventions for the system analytics request, and the conventions that seems to be used throughout as well. This isn't really to remove analytics when the user logs out (I don't think), but more for logging the user out if there is a server error that indicates the user should be logged out.